### PR TITLE
[multistage] Add TPC-H Test for Multistage Engine

### PR DESCRIPTION
**Pull request by ankitsultana**
- [x] Setup tables with very little data
- [x] Stabilize the test
- [x] Add tpch queries

We will tune the dataset in a follow-up PR

cc: @walterddr @kishoreg

### CODE REVIEW

File: pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTest.java [hunk 1]
```diff
@@ -548,7 +548,7 @@ protected void setUpH2Connection()
   protected void setUpH2Connection(List<File> avroFiles)
       throws Exception {
     setUpH2Connection();
-    ClusterIntegrationTestUtils.setUpH2TableWithAvro(avroFiles, getTableName(), _h2Connection);
+    ClusterIntegrationTestUtils.setUpH2TableWithAvro(avroFiles, getTableName(), _h2Connection, getSchema());
```
Comment (walterddr) [inline pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTest.java:551]: looks like base class changes broke some other test please kindly take a look

**Review by walterddr — COMMENTED** [review]
thank you so much for the hard work @ankitsultana 
this is a great effort to setup the framework for tpc-h. 

as of now we can leaf the tuning of the result part-out to a follow up PR. please kindly take a look at my comments.

File: pinot-tools/src/main/java/org/apache/pinot/tools/TPCHQuickStart.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+package org.apache.pinot.tools;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.apache.pinot.tools.admin.PinotAdministrator;
+
+
+public class TPCHQuickStart extends MultistageEngineQuickStart {
```
Comment (xiangfu0) [inline pinot-tools/src/main/java/org/apache/pinot/tools/TPCHQuickStart.java:28]: Let's not create more QuickStart, just add those dataset to MultistageEngineQuickStart.

**Review by xiangfu0 — APPROVED** [review]

File: pinot-tools/src/main/java/org/apache/pinot/tools/TPCHQuickStart.java (hunk 2 above)
Comment (xiangfu0) [reply pinot-tools/src/main/java/org/apache/pinot/tools/TPCHQuickStart.java:28]: Realized that there are a few table name conflicts, let's keep TPCHQuickStart.

**Comment by xiangfu0** [comment]
I'm thinking of making data gen part of the integration test, and allow us to generate larger data sets for joins.
We can make a separated CI check just for this TPC-H integration test.

**Comment by ankitsultana** [comment]
> I'm thinking of making data gen part of the integration test, and allow us to generate larger data sets for joins. We can make a separated CI check just for this TPC-H integration test.

That'd be great. Are you planning to use tpc-h db-gen or are you planning to generate using custom Java code?

**Comment by xiangfu0** [comment]
> > I'm thinking of making data gen part of the integration test, and allow us to generate larger data sets for joins. We can make a separated CI check just for this TPC-H integration test.
> 
> That'd be great. Are you planning to use tpc-h db-gen or are you planning to generate using custom Java code?

Maybe java code is better, which means we can have a separated method in `TPCHQueryIntegrationTest` called `generateDataSet` and do nothing.

Then have a new class `TPCHQueryLargeDataSetIntegrationTest`, which we just override this `generateDataSet` method.

**Pull request merged**